migrations/sql/V2019.09.05.1623__AddSessionSchema.sql

```diff
@@ -0,0 +1,19 @@
+CREATE TABLE user_session
+(
+    session TEXT primary key,
+    user_id TEXT references users (id)
+);
+
+CREATE TABLE file(
+    hash TEXT primary key
+);
+
+CREATE TABLE session_files
+(
+    session   TEXT primary key,
+    file      TEXT,
+    type      TEXT
```

Comment:
should "type" reference an enum table or is this in the app logic?

Reply:
app logic